src/styles/globals.scss

```diff
@@ -1,3 +1,4 @@
-*{
-    font-family: 'Rubik', sans-serif !important;
-}
\ No newline at end of file
+* {
+  font-family: 'Rubik', sans-serif !important;
+  overscroll-behavior: none;
```

Comment:
Is there a reason why you're overriding native scroll effects

Reply:
This is what it looks like with scroll bouncing enabled, so I'm overriding it to stop it from happening
https://github.com/acmucsd/space-2022/assets/27360825/e9e501e9-1493-4ecb-a915-4d4ca6f00076

Reply:
ah I see, getting the gradient out there seems involved